cob_cartesian_controller/src/cartesian_controller.cpp

```diff
@@ -88,12 +91,18 @@ bool CartesianController::initialize()
 	jerk_pub_ = nh_private.advertise<std_msgs::Float64> ("debug/linear_jerk", 1);
 	
 	ROS_WARN("Waiting for Services...");
-	startTracking_ = nh_tracker.serviceClient<cob_srvs::SetString>("start_tracking");
-	stopTracking_ = nh_tracker.serviceClient<std_srvs::Empty>("stop_tracking");
+	startTracking_ = nh_.serviceClient<cob_srvs::SetString>("frame_tracker/start_tracking");
+	stopTracking_ = nh_.serviceClient<std_srvs::Empty>("frame_tracker/stop_tracking");
 	startTracking_.waitForExistence();
 	stopTracking_.waitForExistence();
+
+    action_name_ = "tracking_action";
```

Comment:
you cannot give it the same name as [this action](https://github.com/ipa-fxm/cob_control/blob/indigo_dev/cob_frame_tracker/src/cob_frame_tracker.cpp#L166)
shall we call it `move_cartesian`?